src/vs/vscode.proposed.d.ts

```diff
@@ -167,4 +167,20 @@ declare module 'vscode' {
 		 */
 		export function deleteSecret(service: string, account: string): Thenable<boolean>;
 	}
+
+	export namespace window {
+
+		/**
+		 * Whether the current window is focused.
+		 *
+		 * @readonly
+		 */
+		export let isFocused: boolean;
```

Comment:
Wasn't a typo, tho wasn't thinking. Well, you can have a bit mask or logic like focused implies visible. Tho maybe having an interface is the better options and I pretend is was a typo...